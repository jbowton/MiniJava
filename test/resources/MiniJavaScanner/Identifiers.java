// Should be Identifier
a
pooop
// Should be int literal
5p
