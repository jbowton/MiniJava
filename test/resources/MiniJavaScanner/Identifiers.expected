ID(a) ID(pooop) INTLITERAL(5)